src/Synergism.ts

```diff
@@ -2014,7 +2009,7 @@ export const multipliers = (): void => {
     if (player.achievements[37] > 0.5 && player.prestigePoints.gte(10)) {
         G['globalCrystalMultiplier'] = G['globalCrystalMultiplier'].times(Decimal.log(player.prestigePoints.add(1), 10))
     }
-    if (player.achievements[43] > 0.5) {
+    if (player.achievements[44] > 0.5) {
```

Comment:
is there any indication that player.achievements is 1-based? I couldn't find any in the code

Reply:
I have been extensively checking a lot of things and achievements were always 1 based from what I have seen, an easy way to check this is the Recycle calculation code, a ot of achievements are used there